.jenkins/library/vars/tests.groovy

```diff
@@ -0,0 +1,610 @@
+                if (report_created) {
+                    println("SGX report file created successfully!")
+                } else {
+                    error("Failed to create SGX report file.")
+                }
+                if (evidence_created) {
+                    println("SGX evidence file created successfully!")
+                } else {
+                    error("Failed to create SGX evidence file.")
+                }
+
+                stash includes: 'build/tests/host_verify/host/*.der,build/tests/host_verify/host/*.bin', name: "linux_host_verify-${version}-${build_type}-${BUILD_NUMBER}"
+            }
+        }
+    }
+
+    /* Compile the tests and unstash the certs over for verification.  */
+    stage("Linux nonSGX Verify Quote") {
+        node(globalvars.AGENTS_LABELS["ubuntu-nonsgx"]) {
+            timeout(globalvars.GLOBAL_TIMEOUT_MINUTES) {
+                cleanWs()
+                checkout scm
+                unstash "linux_host_verify-${version}-${build_type}-${BUILD_NUMBER}"
+                def cmakeArgs = "-G Ninja -DBUILD_ENCLAVES=OFF -DCMAKE_BUILD_TYPE=${build_type} -Wdev"
+                def task = """
+                           ${helpers.ninjaBuildCommand(cmakeArgs)}
+                           ctest -R host_verify --output-on-failure --timeout ${globalvars.CTEST_TIMEOUT_SECONDS}
+                           """
+                // Note: Include the commands to build and run the quote verification test above
+                common.ContainerRun("oetools-${version}:${params.DOCKER_TAG}", "clang-10", task, "--cap-add=SYS_PTRACE")
+            }
+        }
+    }
+
+    /* Windows nonSGX stage. */
+    stage("Windows nonSGX Verify Quote") {
+        node(globalvars.AGENTS_LABELS["windows-nonsgx"]) {
+            timeout(globalvars.GLOBAL_TIMEOUT_MINUTES) {
+                cleanWs()
+                checkout scm
+                unstash "linux_host_verify-${version}-${build_type}-${BUILD_NUMBER}"
+                def cmakeArgs = "-G Ninja -DBUILD_ENCLAVES=OFF -DCMAKE_BUILD_TYPE=${build_type} -DNUGET_PACKAGE_PATH=C:/oe_prereqs -Wdev"
+                dir('build') {
+                    bat(
+                        returnStdout: false,
+                        returnStatus: false,
+                        script: """
+                            call vcvars64.bat x64
+                            ${helpers.ninjaBuildCommand(cmakeArgs)}
+                            ctest.exe -V -C ${build_type} -R host_verify --output-on-failure --timeout ${globalvars.CTEST_TIMEOUT_SECONDS} || exit !ERRORLEVEL!
+                        """
+                    )
+                }
+            }
+        }
+    }
+}
+
+def ACCHostVerificationPackageTest(String version, String build_type) {
+    /* Generate an SGX report and two SGX certificates for the host_verify sample.
+    * Also generate and install the host_verify package. Then run the host_verify sample.
+    */
+    stage("ACC-1804 Generate Quote") {
```

Comment:
Are there any plans on running host verify on 20.04?

Reply:
Not at the moment. For another task we should revisit our testing matrix to ensure we catch gaps like this.